test/services/messengerSync/services/schoolSyncService.test.js

```diff
@@ -35,21 +36,18 @@ describe('messenger schoolSync Service', () => {
 			mockery.registerMock('@schul-cloud/commons', commons);
 			mockery.registerMock('../../../utils/rabbitmq', rabbitmqMock);
 			mockery.registerMock('../../utils/rabbitmq', rabbitmqMock);
-			delete require.cache[
-				require.resolve('../../../../src/services/messengerSync/services/schoolSyncService.js')
-			];
 			// eslint-disable-next-line global-require
 			const messengerSync = require('../../../../src/services/messengerSync');
 			app.configure(messengerSync);
```

Comment:
is there really no reload of the app required? which has the mocks finally enabled?